internal/latest/tool_latest_version_finder.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 MongoDB Inc
```

Comment:
given the package name is latest couple of options on naming this file
`latest/lates.go`, `latest/version_finder.go` or `latest/finder.go`

Reply:
oops! yes, I forgot to rename that. thanks for the suggestions